modules/tinymce/src/plugins/importcss/main/ts/core/ImportCss.ts

```diff
@@ -255,7 +257,7 @@ const setup = (editor: Editor): void => {
     };
 
     Tools.each(getSelectors(editor, editor.getDoc(), compileFilter(Settings.getFileFilter(editor))), (selector) => {
-      if (selector.indexOf('.mce-') === -1) {
+      if (selector.match(INTERNAL_EDITOR_STYLE) === null) {
```

Comment:
Should we use `Type.isNull` or `Type.isNullable` from Katamari here?

Reply:
I thought that was only really useful when composing nullable checks 🤔 the standard says it will literally be `null` if there are no matches, so this seemed right.

Reply:
Normally we'd using the `Type` check as it'll compile to smaller code and keeps things consistent.

Reply:
we're talking about 4 bytes difference which will probably be zero after gzip... I don't even need to change it, looking at your later request :)